README.md

```diff
@@ -219,5 +220,22 @@ function toggleHidden(element) {
 	}
 }
 ```
+[↑ back to Table of Contents](#table-of-contents)
+
+## Use `.first()` for single element
+
+### Why?
```

Comment:
split `### Why?` text in summary / background (before Why) and reasons (after Why)? Reasons being: communicate to other devs you expect a single element. More reasons?